community/security/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: single
+title:  "Security"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+## Responsible Disclosure Policy
+If you discover a security vulnerability, the Spinnaker Security SIG would like to know about it. The quicker we know, the quicker we can take steps to address the issue. We ask that you not publicly disclose the vulnerability until we have had the chance to investigate and determine the impact.
```

Comment:
Make the `Spinnaker Security SIG` linkable to the SIG page; or better yet directly to the SIG information table for Security.